python/ray/_private/structured_logging/utils.py

```diff
@@ -0,0 +1,52 @@
+from ray._private.structured_logging.formatters import TextFormatter
+from ray._private.structured_logging.filters import CoreContextFilter
+from typing import Union
+
+LOG_MODE_DICT = {
+    "TEXT": lambda log_level: {
+        "version": 1,
+        "disable_existing_loggers": False,
+        "formatters": {
+            "text": {
+                "()": TextFormatter,
+            },
+        },
+        "filters": {
+            "core_context": {
+                "()": CoreContextFilter,
+            },
+        },
+        "handlers": {
+            "console": {
+                "level": log_level,
+                "class": "logging.StreamHandler",
+                "formatter": "text",
+                "filters": ["core_context"],
+            },
+        },
+        "root": {
+            "level": log_level,
+            "handlers": ["console"],
+        },
+    },
+}
+
+
+class LoggingConfig:
```

Comment:
`@PublicAPI(stability="alpha")`

Also let's move it to job_config.py? since user shouldn't import anything directly from ray/_private

Reply:
Good point. Updated https://github.com/ray-project/ray/pull/45344/commits/dc62d5ce3d79542bb750ed44f8b441e7c112c3ef.